configure.ac

```diff
@@ -17,6 +17,7 @@ AX_ADD_AM_MACRO_STATIC([])
 
 AM_CONDITIONAL(SONIC_ASIC_PLATFORM_BAREFOOT,   test x$CONFIGURED_PLATFORM = xbarefoot)
 AM_CONDITIONAL(SONIC_ASIC_PLATFORM_BROADCOM,   test x$CONFIGURED_PLATFORM = xbroadcom)
+AM_CONDITIONAL(sonic_asic_platform_clounix,   test x$CONFIGURED_PLATFORM = xclounix)
```

Comment:
please keep style the same, make this capital

Reply:
Hi kcudnik：
               The code has been modified to keep the style,please help to review again,thank you !